packages/react-ui/src/widgets/WrapperWidgetUI.js

```diff
@@ -131,27 +131,39 @@ const PaperMenu = styled(Menu)(({ theme }) => ({
   }
 }));
 
-function WrapperWidgetUI(props) {
+const EMPTY_ARRAY = [];
+const DEFAULT_OPTIONS_ICON = <MoreVert />;
+
+function WrapperWidgetUI({
+  title,
+  expanded: _expanded = true,
+  expandable: _expandable = true,
+  onExpandedChange,
+  isLoading = false,
+  disabled = false,
+  options = EMPTY_ARRAY,
+  actions = EMPTY_ARRAY,
+  optionsIcon = DEFAULT_OPTIONS_ICON,
+  children,
+  margin,
+  headerItems,
+  contentProps,
+  footer
+}) {
   const wrapper = createRef();
 
   const [expandedInt, setExpandedInt] = useState(true);
   const externalExpanded =
-    typeof props.expanded === 'boolean' && typeof props.onExpandedChange === 'function';
+    typeof _expanded === 'boolean' && typeof onExpandedChange === 'function';
   const expanded =
-    props.expandable !== false ? (externalExpanded ? props.expanded : expandedInt) : true;
```

Comment:
nit: kinda "refactor" in the one that is really complex and fragile, without typescript and with chance for typo ...
for safety - i would leave props as they were